homeassistant/components/deluge/config_flow.py

```diff
@@ -41,11 +36,8 @@ async def async_step_user(
     ) -> FlowResult:
         """Handle a flow initiated by the user."""
         errors = {}
-        title = None
 
         if user_input is not None:
-            if CONF_NAME in user_input:
-                title = user_input.pop(CONF_NAME)
```

Comment:
I assume these changes are not related to the end of the deprecation and they just a cleanup?

If not, what is the reason for this change to the config flow?

Reply:
It is for handling the configured "name" on import (which is removed, thus this is no longer needed either).